pkg/virt-operator/install-strategy/create.go

```diff
@@ -131,8 +128,13 @@ func createLabelsAndAnnotationsPatch(objectMeta *metav1.ObjectMeta) ([]string, e
 	if err != nil {
 		return ops, err
 	}
+	ownerRefBytes, err := json.Marshal(objectMeta.OwnerReferences)
+	if err != nil {
+		return ops, err
+	}
 	ops = append(ops, fmt.Sprintf(`{ "op": "add", "path": "/metadata/labels", "value": %s }`, string(labelBytes)))
 	ops = append(ops, fmt.Sprintf(`{ "op": "add", "path": "/metadata/annotations", "value": %s }`, string(annotationBytes)))
+	ops = append(ops, fmt.Sprintf(`{ "op": "add", "path": "/metadata/ownerReferences", "value": %s }`, ownerRefBytes))
```

Comment:
If I understand it correctly, we're now setting `ownerReferences` to `null` for objects w/o the reference (which are all our objects now...) wouldn't it cause NPE in K8S' validation?
https://github.com/kubernetes/kubernetes/blob/de1cfa9bc16a08116b8f6b56929d3b1ea39fd1e3/staging/src/k8s.io/apimachinery/pkg/api/validation/objectmeta.go#L83

Reply:
@danielBelenky first it is serialized and deserialized, second it is possible to iterate over a `nil` list. It has the same behaviour like iterating over an empty list. So there are no issues to expect. But nice that you looked up the validation code :+1: